src/api/che.ts

```diff
@@ -139,6 +140,34 @@ export class CheHelper {
     return destinaton
   }
 
+  /**
+   * Retreives Keycloak admin user credentials.
+   * Works only with installers which use Che CR (operator, olm).
+   * Returns credentials as an array of two values: [login, password]
+   * In case of an error an array with undefined values will be returned.
+   */
+  async retrieveKeycloakAdminCredentials(cheNamespace: string): Promise<string[]> {
+    let adminUsername
+    let adminPassword
+
+    const cheCluster = await this.kube.getCheCluster('eclipse-che', cheNamespace)
```

Comment:
Use the constant
https://github.com/che-incubator/chectl/blob/master/src/constants.ts#L25